doc/api/stream.md

```diff
@@ -369,12 +369,17 @@ See also: [`writable.uncork()`][].
 added: v8.0.0
 -->
 
-* `error` {Error}
+* `error` {Error} Optional, an error to emit with `'error'` event.
```

Comment:
There is no need to explicitly list it as "Optional". That's communicated by `[error]` in the signature on line 367.

Reply:
Errors are not emitted. Events are emitted. Maybe: ```Provided as the argument to callbacks listening for the `error` event``` or something like that?

Reply:
> There is no need to explicitly list it as "Optional". That's communicated by `[error]` in the signature on line 367.

Ugh, never mind that comment. I see "Optional" is all over this doc already. So, yeah, consistency over my preferences for sure.